dist/docker-images/ziti-router/Dockerfile

```diff
@@ -37,4 +32,25 @@ RUN chown -R ziggy:ziggy /ziti-router
 # run as ziggy (2171:2171) by default, override run-as user with root when DOCKER_ROUTER_MODE=tproxy
 USER ziggy
 
+# configure logging
+ENV PFXLOG_NO_JSON=true
+ENV ZITI_TIME_FORMAT=utc
+
+# globally toggle bootstrapping; false preserves "thin" behavior for existing deployments like K8s
+ENV ZITI_BOOTSTRAP=false
```

Comment:
bootstrapping is disabled by default in the image to preserve existing "thin" deployments, e.g., k8s